data/jibauni/jibauni missions.txt

```diff
@@ -0,0 +1,579 @@
+			heroic
+			merciful
+			opportunistic
+			plunders
+			unconstrained
+			confusion 9 # these are specially trained Jibauni
+		ship "Nathar" "Muna Wasa"
+
+event "ruwin: merchant reputation back to normal"
+	"reputation: Merchant" += 1000
+
+
+mission "Jibauni: Ji-wi"
+	minor
+	name "Take Ji-wi to <stopovers>"
+	description "Ji-wi the Jibauni would like you to take him to <stopovers> and then back to <planet>."
+	source "Kahbani"
+	stopover "Baitoon"
+	destination "Kahbani"
+	to offer
+		random < 15
+	on offer
+		conversation
+			`You notice a Jibauni running between captains, every now and then pausing to ask something. You notice the Jibauni turn towards you and approach at a resolute pace.`
+			choice
+				`	(Wait for the Jibauni to reach you.)`
+				`	(Leave the area.)`
+					decline
+			`	As you wait, you see the Jibauni pause to speak to other captains. But nothing comes of those encounters, and the Jibauni finally gets close to you.`
+			choice
+				`	(Wait for it to say something.)`
+				`	"My name is <first> of the <ship>. How's it going?"`
+			`	"My name is Ji-wi," announces the Jibauni. "Can you take me to <stopovers> and then bring me back here? I will pay you <payment>."`
+			choice
+				`	"I'd be happy to take you."`
+				`	"Sorry, no deal."`
+					decline
+			`	"Good! I will be ready and at your ship by the time you get back to it," says Ji-wi. He then leaves.`
+				accept
+	on stopover
+		dialog `Ji-wi is only gone for a moment before returning and asking that you go back to <planet> soon.`
+	on complete
+		payment 1500
+		conversation
+			`"Thank you for transporting me to the capital and back," says Ji-wi, paying you <payment>. He then walks off into the spaceport of <origin>, answering no questions whatsoever about this trip.`
+
+
+mission "Jibauni: Dran-lin 0"
+	minor
+	name "Take Dran-lin to <planet>"
+	description "Dran-lin would like you to take him to <destination>, to see whether moving to Rebel Jibauni worlds is a good idea."
+	passengers 1
+	source
+		government "Jibauni"
+	destination
+		government "Jibauni (Rebel)"
+	to offer
+		random < 45
+		not "event: jibauni rebel jibauni peace"
+	on offer
+		conversation
+			`As you are walking through the spaceport, you collide with a younger Jibauni. He immediately stands up and apologizes, then notices who you are. "Oh, the strange alien! Hello! May I ask of you a favor?"`
+			choice
+				`	"I guess."`
+				`	"Sorry for bumping into you. Sure you can ask!"`
+				`	"Nope, not right now. Other things to do."`
+					goto decline
+			`	"Would you be willing to transport me, Dran-lin, to the planet of <planet>? I would be interested in seeing the Rebel Jibauni and how they operate. I may end up joining them, or else I will ask you to take me back to a Jibauni planet."`
+			choice
+				`	"You know, the Rebel Jibauni are bad creatures. Are you sure you want to go?"`
+					goto rebelsbad
+				`	"The wonderful Rebel Jibauni? Sure, I'll take you immediately!"`
+					goto rebelsgood
```

Comment:
I am thinking there could be a neutral response here as well? Maybe you don't feel too strongly about the rebels either way. A job's a job.